projects/avodata/AvodataSolverTool.java

```diff
@@ -0,0 +1,142 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Scanner;
+
+import org.apache.commons.math3.analysis.function.Abs;
+import org.apache.commons.math3.linear.Array2DRowRealMatrix;
+import org.apache.commons.math3.linear.DecompositionSolver;
+import org.apache.commons.math3.linear.MatrixUtils;
+import org.apache.commons.math3.linear.RealMatrix;
+import org.apache.commons.math3.linear.RealVector;
+import org.apache.commons.math3.linear.SingularValueDecomposition;
+import org.apache.commons.math3.util.CombinatoricsUtils;
+
+/** To build: bazel build //projects/avodata:ls-alg_tool To run: bazel-bin/projects/avodata/ls-alg_tool */
+/** This class reads matrix from csv file, ignores the parmeters and calculate 
+/** using least squares */
+public class AvodataSolverTool {
+
+  static final String fpath = "/projects/avodata/check.csv";
```

Comment:
static finals are constants, so are `UPPER_CASE`.
Also, please use full names (e.g `FILE_PATH`).
I would actually change this to `CSV_PATH` to be more descriptive.